packages/card/src/Card.tsx

```diff
@@ -24,25 +25,27 @@ interface ColorSet {
   clickableStyle: string;
 }
 
-const lightBaseBoxShadow = `0 4px 10px -4px ${transparentize(
-  0.7,
-  uiColors.black,
-)}`;
+// prettier-ignore
+const lightBaseBoxShadow = `0 4px 10px -4px ${transparentize(0.7, '#000')}`;
```

Comment:
Looks like the color should be `rgba(0, 30, 43, 0.3)` which is `palette.black`

Reply:
Ah, yeah there was a discrepancy between the Figma files here